pytorch_lightning/callbacks/lr_monitor.py

```diff
@@ -192,6 +209,23 @@ def _add_prefix(
         count = seen_optimizer_types[optimizer_cls]
         return name + f'-{count - 1}' if count > 1 else name
 
+    def _add_suffix(self, name: str, param_groups: List[Dict], param_group_index: int, use_names: bool = True) -> str:
```

Comment:
nit: do you see value in the full explicit name?

```suggestion
    def _add_suffix(self, name: str, param_groups: List[Dict], index: int, use_names: bool = True) -> str:
```

Reply:
My experience, I think, biases me to explicit names, especially for `kwargs` in particular. But, as I am a guest here I am happy to defer to the collective's preferences.